contracts/BC_fusion/StakeHub.sol

```diff
@@ -595,16 +605,21 @@ contract StakeHub is System, Initializable {
     /**
      * @dev Malicious vote slash. Only the `SlashIndicator` contract can call this function.
      */
-    function maliciousVoteSlash(bytes calldata _voteAddr) external onlySlash {
-        address operatorAddress = _voteToOperator[_voteAddr];
+    function maliciousVoteSlash(bytes calldata voteAddress) external onlySlash {
+        address operatorAddress = voteToOperator[voteAddress];
         if (!_validatorSet.contains(operatorAddress)) revert ValidatorNotExist(); // should never happen
         Validator storage valInfo = _validators[operatorAddress];
 
-        uint256 index = block.timestamp / BREATH_BLOCK_INTERVAL;
+        uint256 index = block.timestamp / BREATHE_BLOCK_INTERVAL;
         // This is to prevent many honest validators being slashed at the same time because of implementation bugs
-        if (_felonyMap[index] >= felonyPerDay) revert NoMoreFelonyToday();
+        if (_felonyMap[index] >= maxFelonyBetweenBreatheBlock) revert NoMoreFelonyAllowed();
         _felonyMap[index] += 1;
 
+        // check if the voteAddress has already expired
+        if (voteExpiration[voteAddress] != 0 && voteExpiration[voteAddress] < block.timestamp) {
```

Comment:
If the slash event happens within one day before it is expired, it still can be slashed.